tests/test_openclhpp.cpp

```diff
@@ -3378,4 +3378,47 @@ void testCommandBufferInfoKHRCommandQueues()
 #endif
 }
 
+/****************************************************************************
+ * Tests for allocate and deallocate API (clSVMAlloc, clSVMFree, clEnqueueSVMMap)
+ ****************************************************************************/
+
+int *testMemory;
```

Comment:
Is there any way to do this test without needing a global variable?  I'm not going to lose a ton of sleep over it, but if we ever want to enable parallel test execution it will be important not to have global variables, and we've been pretty good about avoiding global variables so far.

Does this test even need to allocate any real memory?  If not, can we just return one of the e.g. platform handles (`make_platform_id(0)`)?